src/components/views/settings/tabs/user/GeneralUserSettingsTab.js

```diff
@@ -26,6 +27,7 @@ import LanguageDropdown from "../../../elements/LanguageDropdown";
 import AccessibleButton from "../../../elements/AccessibleButton";
 import DeactivateAccountDialog from "../../../dialogs/DeactivateAccountDialog";
 import PropTypes from "prop-types";
+import {SUPPORTED_THEMES} from "../../../../../settings/controllers/ThemeController";
```

Comment:
Setting controllers aren't supposed to be exposed outside of the settings package, however I don't exactly fancy forcing us through hoops just because we can make hoops. 

Ideally the settings interface would expose a `getOptions` function or similar, however that seems a bit overkill for the one place that could use it.

Reply:
broke it out a different way, so they both depend on a common file